sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MicroBatchExecution.scala

```diff
@@ -508,12 +508,20 @@ class MicroBatchExecution(
           cd.dataType, cd.timeZoneId)
     }
```

Comment:
The root cause is `CurrentBatchTimestamp` is `TimeZoneAwareExpression` which is unresolved without TimeZoneId.